cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/SQLEngineCollection.java

```diff
@@ -232,7 +231,7 @@ public boolean tryStoreDirect(StageSpec stageSpec) {
     Set<String> directStoreSinks = new HashSet<>();
 
     // Create list to store all tasks.
-    List<Future<String>> directStoreFutures = new ArrayList<>(sinks.size());
+    List<CompletableFuture<String>> directStoreFutures = new ArrayList<>(sinks.size());
```

Comment:
Any specific reason for this changes? Any `CompletableFuture` is a `Future`

Reply:
Just so we can match the returned type perfectly. I guess it's not really necessary at this stage.